modules/core/src/v2/environments.ts

```diff
@@ -193,30 +173,6 @@ export const Environments: Environments = {
     zecExplorerBaseUrl: 'https://explorer.testnet.z.fakeurl/api',
     dashExplorerBaseUrl: 'https://testnet-insight.dashevo.fakeurl/insight-api',
   }),
-  rmgLocal: Object.assign({}, devBase, {
-    uri: 'https://rmglocalhost:3000',
-    stellarFederationServerUrl: 'https://rmglocalhost:3000/api/v2/txlm/federation',
-  }),
-  rmglocalNonSecure: Object.assign({}, devBase, {
-    uri: 'http://rmglocalhost:3000',
-    stellarFederationServerUrl: 'http://rmglocalhost:3000/api/v2/txlm/federation',
-  }),
-  msProd: Object.assign({}, mainnetBase, {
-    uri: 'https://app.bitgo.com',
-    stellarFederationServerUrl: 'https://app.bitgo.com/api/v2/xlm/federation',
-  }),
-  msTest: Object.assign({}, testnetBase, {
-    uri: 'https://app.bitgo-test.com',
-    stellarFederationServerUrl: 'https://app.bitgo-test.com/api/v2/txlm/federation',
-  }),
-  msDev: Object.assign({}, devBase, {
-    uri: 'https://app.bitgo-dev.com',
-    stellarFederationServerUrl: 'https://app.bitgo-dev.com/api/v2/txlm/federation',
-  }),
-  msLatest: Object.assign({}, devBase, {
-    uri: 'https://app.bitgo-latest.com',
-    stellarFederationServerUrl: 'https://app.bitgo-latest.com/api/v2/xlm/federation',
-  }),
```

Comment:
Probably needs to keep: https://docs.google.com/document/d/1ef89ZmswlN7rVQMZfvbUhDG6w8XlkmF8PGyrJxzJVIg/edit

in mind